extensions/objects/templates/ListOfSetsOfHtmlStringsEditor.js

```diff
@@ -76,7 +76,7 @@ oppia.directive('listOfSetsOfHtmlStringsEditor', [
 
         $scope.selectedItem = function(choiceListIndex, selectedRank) {
           var choiceHtml = $scope.choices[choiceListIndex].id;
-          var selectedRank = parseInt(selectedRank) - 1;
+          selectedRank = parseInt(selectedRank) - 1;
```

Comment:
Better to give this (or the input arg) another name. Probably the input arg should be renamed to make clear it's a string(?)

Reply:
Done.